website/content/en/docs/migration/version-upgrade-guide.md

```diff
@@ -1365,12 +1365,12 @@ first `COPY` from `COPY /*.yaml manifests/` to `COPY deploy/olm-catalog/<operato
 [mercurial]: https://www.mercurial-scm.org/downloads
 [migrating-to-modules]: https://github.com/golang/go/wiki/Modules#migrating-to-modules
 [modules-wiki]: https://github.com/golang/go/wiki/Modules#migrating-to-modules
-[print-deps-cli]: /docs/cli/operator-sdk_print-deps
+[print-deps-cli]: https://v0-19-x.sdk.operatorframework.io/docs/cli/operator-sdk_print-deps/
 [changelog]: https://github.com/operator-framework/operator-sdk/blob/master/CHANGELOG.md
 [release-notes]: https://github.com/operator-framework/operator-sdk/releases
 [v0.1.0-migration-guide]: ../v0.1.0-migration-guide
 [manifest-format]: https://github.com/operator-framework/operator-registry#manifest-format
-[client-doc]: ../../golang/legacy/references/client
+[client-doc]: https://v0-19-x.sdk.operatorframework.io/docs/golang/legacy/references/client/
```

Comment:
Good spot since it was removed from the master we need to point out to the latest release with them 👍